pkgs/games/openrw/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, cmake, sfml, mesa, bullet, glm, libmad, x11 }:
+
+let
+  inih = fetchFromGitHub {
+    owner = "benhoyt";
+    repo = "inih";
+    rev = "0c3f8ea80ca2c8be8995a0b7b4e84af1641a4dd2";
+    sha256 = "0w0xqcxkyhnqg1q1iwxkaczkwcmj7z9b9qkr3580gvi50nhmkssy";
+  };
+in
+stdenv.mkDerivation rec {
+  version = "2016-06-29";
+  name = "openrw-${version}";
+  src = fetchFromGitHub {
+    owner = "rwengine";
+    repo = "openrw";
+    rev = "46a7254a41d9f6e1eda8d31e742de492abb2540e";
+    sha256 = "0l224ij3askn8rhpm21y8wnmwmpvwy1p60c0vhnlqrf320pcgz7b";
+  };
+
+  buildInputs = [ cmake inih sfml mesa bullet glm libmad x11 ];
+
+  INSTALL_PATH = "$(out)";
+
+  postUnpack = ''
+    rmdir $sourceRoot/external/inih
+    cp -r ${inih} $sourceRoot/external/inih
+    chmod 755 -R $sourceRoot/external/inih
```

Comment:
It may be better to use `fetchgit` with `fetchSubmodules = true;` (default) instead of manual download and replacing.